packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -40,7 +41,9 @@ export const styles = {
       cursor: 'default',
     },
   },
+  /* Styles applied to the root element if `disabled={true}`. */
```

Comment:
I think that we are going to need a specific wording for the "internal state" rules: https://material-ui.com/customization/overrides/#overriding-with-classes

Reply:
Maybe we could have a `Type` column with 'modifier | internal state'.